docs/expected-to-fail-tests.adoc

```diff
@@ -44,6 +44,18 @@ A custom message can be provided, explaining why the tested code is not working
 include::{demo}[tag=expected_to_fail_message]
 ----
 
+In some cases it may be desirable to only have the test be aborted on certain exceptions.
```

Comment:
I like the documentation change but the project adheres to the "one-sentence-per-line" rule. Could you format it that way, please?

Reply:
I should have read that more carefully. Sorry about that!